include/rdma/fi_ext.h

```diff
@@ -41,6 +42,24 @@
 extern "C" {
 #endif
 
+/*
+ * Each provider can define provider-specific values by
+ * choosing an unique 11-bit (since the most significant
+ * bit must be 1) provider-specific code to avoid
+ * overlapping with other providers. E.g.,
```

Comment:
I think the comment can be adjusted to make thing clearer. 

My understanding is: each provider define an unique mask, each provider specific option must contain this tag.

The tag's format is:

The highest bit (31st bit) must be toggled on, which is FI_PROV_SPECIFIC

bit 20 to 30 must be unqiue for each provider.

bit 0 to 20 is used by each option.

Reply:
Thanks for your review, @wzamazon. Just saw it after it's merged. 
We made further change to this part of code in #7084, pls let me know if it looks okay.